test/check_event_log.txt

```diff
@@ -0,0 +1,18 @@
+; Checks the pattern of event log file. Makes sure the data rows
+; Corresponding to each layer have the correct format.
+; Checks the shader hashes across different rows and guarantees they are
+; consistent.
+; Counts the number of memory and frame time logs and check if they are
+; as expected.
+; CHECK-DAG: compile_time_layer_init,{{[0-9]+}}
+; CHECK-DAG: memory_usage_layer_init,{{[0-9]+}}
+; CHECK-DAG: frame_time_layer_init,{{[0-9]+}}
+; CHECK: create_shader_module_ns,{{[0-9]+}},[[SHADER1:0x[a-zA-Z0-9]+]],{{[0-9]+}}
```

Comment:
nit: we could add spaces after `:` to pad the start of each CSV line. FileCheck ignores most spaces, so it's common to format checks for readability. For example:
```
; CHECK:      blah
; CHECK-NEXT: foo